mitodl/mitxpro
# Added API call to create edX user when xpro user is created

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #68

#### What's this PR do?
Adds API call to create edX user when xpro user is created

#### How should this be manually tested?
Add a user in xpro (Django admin, shell, etc) while running LMS. Check to make sure the user is also created in LMS

#### Any background context you want to provide?
I decided on a data migration to ensure the existence of the OAuth application since the app really won't function correctly without it.
